File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

for (let i = 1; i <= parseInt(input[0]); i++) {
  const [A, B] = input[i].split(" ");
  console.log(parseInt(A) + parseInt(B));
}

File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.swift
let N: Int = Int(readLine()!)!
var inputs: [[Int]] = []
for _ in 0 ..< N {
    inputs.append(readLine()!.split(separator: " ").map { Int($0)! })
}

for input in inputs {
    print(input[0] + input[1])
}